CMakeLists.txt

```
@@ -249,17 +250,25 @@ endif()
 check_library_exists(edit readline "" HAVE_EDIT)
 find_opt_library_and_header(EDIT_PATH histedit.h EDIT_LIB edit HAVE_EDIT)
 
-#find_package(Gettext)           # Used for running tests
+find_package(Gettext ${Required_Gettext_Version})
+find_package(Intl)
 
-#if (GETTEXT_FOUND)
-#  set(HAVE_GETTEXT 1)
-#else()
+if(GETTEXT_FOUND AND Intl_FOUND)
+  set(GETTEXT_DOMAIN "${PROJECT_NAME}")
+  include(GNUInstallDirs)
+  if(CMAKE_INSTALL_LOCALEDIR)
+    set(GETTEXT_LOCALE_DIR "${CMAKE_INSTALL_LOCALEDIR}")
+  else()
+    set(GETTEXT_LOCALE_DIR "FIXME")
```

What is a sensible default here?
```suggestion
    set(GETTEXT_LOCALE_DIR "locale")
```